hack/build-go.sh

```diff
@@ -32,7 +32,13 @@ fi
 
 mkdir -p ${BIN_PATH}
 
+# Need to enable CGO and use the containers_image_opengpg flag because we are modifying the
+# /etc/containers/policy.json file whenever allowed/blocked registries are added
 CGO_ENABLED=0
```

Comment:
But this is _disabling_ CGO :) Also `s/gpg/pgp/`.

Maybe something like
> Use the containers_image_openpgp` flag to avoid the default CGO implementation of signatures dragged in by containers/image/signature, which we use only to edit the /etc/containers/policy.json file without doing any cryptography

Reply:
woops, I overlooked the 0. Fixed now